CONTRIBUTING.md

```diff
@@ -34,6 +34,14 @@ FYI if you use macOS and [Homebrew] you can install it by
 Make sure that your .NET Core SDK is 3.1 or higher.  You could show
 the version you are using by `dotnet --info` command.
 
+{% warning %}
```

Comment:
What's this syntax? Is it a part of GFM?

Reply:
I've misunderstood. it's introduced on [GitHub Docs Contribution Guide](https://github.com/github/docs/blob/f3bebc9da4b546cd4d2d111e72e8838b494c4dea/contributing/content-markup-reference.md#callout-tags) but it doesn't seemed to GFM😅